doc/concepts/configuration/configuration_etcd.rst

```diff
@@ -1,94 +1,228 @@
+        :language: yaml
+        :start-at: groups:
+        :end-at: 127.0.0.1:4403
+        :dedent:
+
+2.  :ref:`Create a user <configuration_credentials_managing_users_roles>` and grant them the following privileges:
+
+    *   The ``read`` and ``write`` permissions to the ``config_storage`` and ``config_storage_meta`` spaces used to store configuration data.
+    *   The ``execute`` permission to ``universe`` to allow interacting with the storage using the tt utility.
+
+    ..  literalinclude:: /code_snippets/snippets/centralized_config/instances.enabled/tarantool_config_storage/config.yaml
+        :language: yaml
+        :start-at: credentials:
+        :end-before: replicator:
+        :dedent:
+
+3.  Set the :ref:`replication.failover <configuration_reference_replication_failover>` option to ``election`` to enable automated failover:
+
+    ..  literalinclude:: /code_snippets/snippets/centralized_config/instances.enabled/tarantool_config_storage/config.yaml
+        :language: yaml
+        :start-at: replication:
+        :end-at: failover: election
+        :dedent:
+
+4.  Enable the :ref:`MVCC transaction mode <txn_mode_transaction-manager>` to provide linearizability of read operations:
+
+    ..  literalinclude:: /code_snippets/snippets/centralized_config/instances.enabled/tarantool_config_storage/config.yaml
+        :language: yaml
+        :start-at: database:
+        :end-at: use_mvcc_engine
+        :dedent:
+
+The resulting storage configuration might look as follows:
+
+..  literalinclude:: /code_snippets/snippets/centralized_config/instances.enabled/tarantool_config_storage/config.yaml
     :language: yaml
+    :end-before: Interact with config storage
     :dedent:
 
--   :ref:`config.etcd.endpoints <config_etcd_endpoints>` specifies the list of etcd endpoints.
--   :ref:`config.etcd.prefix <config_etcd_prefix>` sets a key prefix used to search a configuration. Tarantool searches keys by the following path: ``<prefix>/config/*``. Note that ``<prefix>`` should start with a slash (``/``).
+You can find the full example here: `tarantool_config_storage <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/centralized_config/instances.enabled/tarantool_config_storage>`_.
 
-You can also provide additional etcd connection options.
-In the example below, the following options are configured in addition to an endpoint and key prefix:
 
-..  literalinclude:: /code_snippets/snippets/config/instances.enabled/etcd_full/config.yaml
-    :language: yaml
+.. _centralized_configuration_storage_tarantool_start:
+
+Starting a storage
+******************
+
+To start instances of the configured storage, use the ``tt start`` command, for example:
+
+.. code-block:: console
+
+    $ tt start tarantool_config_storage
+
+Learn more from the :ref:`Starting and stopping instances <admin-start_stop_instance>` section.
+
+
+
+.. _centralized_configuration_storage_tarantool_interact:
+
+Interacting with the storage
```

Comment:
BTW, `config.storage` can also be used to publish a cluster's configuration :) But I agree that this might be not the best place. As an idea, I can drop the `Interacting with the storage` and `Watching configuration updates` sections, move the net.box-related content to a reference page and add a note inside `Tarantool-based storage` with links to reference pages.